src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,14 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine excavator = new Excavator();
+        Machine bulldozer = new Bulldozer();
+        Machine[] machines = new Machine[] {truck, excavator, bulldozer};
+        for (Machine vehicle : machines) {
```

Comment:
```suggestion
        for (Machine machine : machines) {
```
Better do not mix names. You may create class Vehicle in the future.